File: ChatBot Task/Chatbot.js
function sendMessage() {
    var userInput = document.getElementById("userInput").value;
    var chatbox = document.getElementById("chatbox");

    if (userInput.trim() === "") {
        return;
    }

    // Add user message to chatbox
    var userMessage = `<div class="message user"><div class="avatar"></div><div class="text">${userInput}</div></div>`;
    chatbox.innerHTML += userMessage;

    // Clear the input field
    document.getElementById("userInput").value = "";

    // Process user input
    var lowerCaseInput = userInput.toLowerCase();
    var response = "";

    if (lowerCaseInput === "hi" || lowerCaseInput === "hello" || lowerCaseInput === "hey") {
        response = "Hello! How can I assist you?";
    } else if (lowerCaseInput.includes("your name")) {
        response = "I am a simple chatbot. I am here to assist you.";
    } else if (lowerCaseInput.includes("how are you")) {
        response = "I am just a bunch of code, but I can help you!";
    } else if (lowerCaseInput.includes("translate this in")) {
        response = "I can't do that, but I can help you with other queries.";
    } else if (lowerCaseInput === "bye") {
        response = "Goodbye! Have a nice day!";
    } else if (lowerCaseInput.includes("time")) {
        response = "I can't tell the time, but you can check your device for the current time.";
    } else if (lowerCaseInput.includes("help")) {
        response = "I am here to help you with some basic inputs. You can say 'hi', ask about my name, or say 'bye' to end the conversation.";
    } else {
        response = "I'm sorry, I don't understand that. Please type again clearly.";
    }

    // Add bot response to chatbox
    var botMessage = `<div class="message bot"><div class="avatar"></div><div class="text">${response}</div></div>`;
    chatbox.innerHTML += botMessage;

    // Scroll to the bottom of the chatbox
    chatbox.scrollTop = chatbox.scrollHeight;
}

File: chatbot.cpp
#include<iostream>
#include<string>
using namespace std;

int main(){
    cout<<"Hello! I am SIRI. How can I help you?"<<endl;

    while(true)
    {
        cout<<"You: ";
        string  input;
        getline(cin, input);

        for(char &c: input)
        {
            c=tolower(c);
        }

        if(input == "hi" || input == "hello" || input == "hey")
        {
            cout<<"SIRI: Hello! How can I assist you?"<<endl;
        }
        else if(input.find("your name") != string::npos)
        {
            cout<<"SIRI: I am a simple chatbot. I am here to assist you."<<endl;
        }
        else if(input.find("how are you") != string::npos)
        {
            cout<<"I am just a bunch of code, but i can help you!"<<endl;
        }
        else if(input.find("translate this in") != string::npos)
        {
            cout<<"SIRI: I can't do this, but I can help you."<<endl;
        }
        else if(input == "bye")
        {
            cout<<"SIRI: Goodbye! Have a nice day!"<<endl;
            break;
        }
        else if(input.find("time") != string::npos)
        {
            cout<<"SIRI: I can't tell time, but you can check your device for current time."<<endl;
        }
        else if(input.find("help") != string::npos)
        {
            cout<<"SIRI: I am here to help you with some basic inputs, you can say 'hi', ask about my name, or say 'bye' to end conversation."<<endl;
        }
        else{
            cout<<"SIRI: I'm sorry, I don't understand that. Please type again clearly."<<endl;
        }
    }
}